src/dummyData/rolemanagement.json

```diff
@@ -0,0 +1,70 @@
+[
+    {
+      "name": "Keza Alice",
+      "email":"keza@gmail.com",
+      "status": "active",
+      "role": "admin" ,
+      "value":"1" 
```

Comment:
What is the use of the value attribute in the object.

It may not be related to the PR but if this value is not related to the user model, you can safely remove it.